src/utils/serialization/VectorBufferStream.ts

```diff
@@ -0,0 +1,223 @@
+export class VectorBufferStream {
+    private buffer: Buffer
```

Comment:
Use VectorBufferStream from shardus core. Request you to test out the chenges in a local nw